models/intel/time-series-forecasting-electricity-0001/accuracy-check.yml

```diff
@@ -0,0 +1,30 @@
+models:
+  - name: time-series-forecasting-electricity-0001
+    launchers:
+      - framework: dlsdk
+        device: CPU
+        inputs:
+          - name: timestamps
+            type: INPUT
+            layout: NHWC
+        adapter:
+          type: quantiles_predictor
+          output_name: quantiles
+          quantiles:
+            0.5: 1
+            0.9: 2
+
+    datasets:
+      - name: Electricity
+        annotation_conversion:
+          converter: electricity
+          data_path_file: LD2011_2014.txt
+        reader:
+          type: annotation_features_extractor
+          features:
+            - inputs
```

Comment:
please move annotation conversion to data/dataset)definitions.yml

Reply:
done. Is it good for you now?)

Reply:
yes, not it looks good :)